README.md

```diff
@@ -26,13 +26,15 @@ After installing these dependencies, the latest stable version of cocotb can be
 pip install cocotb
 ```
 
-For more details, including how to install a development version of cocotb, see [the documentation](https://docs.cocotb.org/en/latest/install.html).
+:warning: **Windows Users** See [here](https://docs.cocotb.org/en/latest/install.html) for installation instructions.
+
+For more details, including how to install a development version of cocotb, see [the documentation](https://docs.cocotb.org/en/latest/install_devel.html#install-devel).
```

Comment:
```suggestion
For more details, including how to install a development version of cocotb, see [the documentation](https://docs.cocotb.org/en/stable/install_devel.html#install-devel).
```
?

Reply:
That link doesn't work. It's something else that will have to be updated after the release.

Reply:
We need to update it _before_ the release, because the readme ends up snapshotted on PyPI.

Reply:
You're right, it's okay if it's broken for a little bit, right?

Reply:
Much better than being broken for a whole release cycle.

Reply:
Actually I'm not sure if this should change. If we want to offer users the latest version, the instructions for how to install the latest dev version may not be correct if we reference stable.

The above line for Windows users should change though and will be broken until 1.4 is released.

Reply:
Unfortunately, this sentence is both about:

* More details (stable)
* Development version (latest)

Perhaps we should have
* This link to stable
* The development section of stable link to latest.

Reply:
I split the sentence in two. One reference stable for general "more info" and the other for documentation on installing the development version, which will deposit them in the right context to browse the rest of the development documentation.